authenticate/management/commands/createusers.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+from authenticate.models import CustomUser
+from django.utils.crypto import get_random_string
+
+class Command(BaseCommand):
+    help = 'Create Users'
+
+    def add_arguments(self, parser):
+        parser.add_argument('users', type=int, help='Create n number of Users')
+
+    def handle(self, *args, **options):
+      
+      for UserNo in range(options['users']):

```

Comment:
In python we use underscore instead of camel case. `UserNo` -> `user_number`

Reply:
Done